Profile provider compilation failures

Appveyor:

```
error[E0599]: no method named `unwrap` found for type `profile::ProfileProviderFuture` in the current scope
   --> rusoto\credential\src\profile.rs:388:26
    |
388 |         assert_eq!(creds.unwrap().aws_access_key_id(), "bar_access_key");
    |                          ^^^^^^
error: aborting due to previous error
error: Could not compile `rusoto_credential`.
Caused by:
...
```

TravisCI:

```
error[E0599]: no method named `unwrap` found for type `profile::ProfileProviderFuture` in the current scope
   --> rusoto/credential/src/profile.rs:388:26
    |
388 |         assert_eq!(creds.unwrap().aws_access_key_id(), "bar_access_key");
    |                          ^^^^^^
error: aborting due to previous error
error: Could not compile `rusoto_credential`.
```

Probably leftover from the recent async changes and modifications to the credentials crate.

--- a/rusoto/credential/src/profile.rs
+++ b/rusoto/credential/src/profile.rs
@@ -384,7 +384,7 @@ mod tests {
         assert!(result.is_ok());
         let provider = result.unwrap();
         assert_eq!(provider.file_path().to_str().unwrap(), credentials_path);
-        let creds = provider.credentials();
+        let creds = provider.credentials().wait();
         assert_eq!(creds.unwrap().aws_access_key_id(), "bar_access_key");
         env::remove_var("AWS_SHARED_CREDENTIALS_FILE");
         env::remove_var("AWS_PROFILE");

